docs/installation/debian.md

```diff
@@ -108,7 +108,7 @@ exit
 `cd /var/www/monica` then run these steps with `sudo`:
 
 1. `cp .env.example .env` to create your own version of all the environment variables needed for the project to work.
-1. Update `.env` to your specific needs. Don't forget to set `DB_USERNAME` and `DB_PASSWORD` with the settings used behind. You'll need to configure a [mailserver](/docs/installation/mail.md) for registration & reminders to work correctly.
+1. Update `.env` to your specific needs. Don't forget to set `DB_USERNAME` and `DB_PASSWORD` with the settings used behind. You'll need to configure a [mailserver](/docs/installation/mail.md) for registration & reminders to work correctly. Also you need to set the `APP_ENV` variable to "*production*". "*local*" is only used for the development version.
```

Comment:
It might be better like this:
```suggestion
1. Update `.env` to your specific needs
   - set `DB_USERNAME` and `DB_PASSWORD` with the settings used behind.
   - configure a [mailserver](/docs/installation/mail.md) for registration & reminders to work correctly.
   - set the `APP_ENV` variable to `production`, `local` is only used for the development version.
```